<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Results</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>461</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seven Lions</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Seven Lions.jpg</normaloff>:/images/Seven Lions.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Excision</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Excision.jpg</normaloff>:/images/Excision.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Metallica</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Metallica.png</normaloff>:/images/Metallica.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Petit Biscuit</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/PetitBiscuit.png</normaloff>:/images/PetitBiscuit.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K?D</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Kid.jpg</normaloff>:/images/Kid.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Three Days Grace</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/3DG.png</normaloff>:/images/3DG.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barely Alive</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Barely-Alive.png</normaloff>:/images/Barely-Alive.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zomboy</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Zomboy.png</normaloff>:/images/Zomboy.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Herobust</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Herobust.png</normaloff>:/images/Herobust.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pink Floyd</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Pink-Floyd-Logo.png</normaloff>:/images/Pink-Floyd-Logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linkin Park</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Linkin-Park-Logo.png</normaloff>:/images/Linkin-Park-Logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Snails</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Snails-logo.png</normaloff>:/images/Snails-logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Papa Roach</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Papa_Roach_Logo.png</normaloff>:/images/Papa_Roach_Logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tiesto</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Tiësto_logo.png</normaloff>:/images/Tiësto_logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Boogie T</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/BoogieT_Logo.png</normaloff>:/images/BoogieT_Logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Above &amp; Beyond</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Above_&amp;_Beyond_Logo.png</normaloff>:/images/Above_&amp;_Beyond_Logo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spag Heddy</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/SpagLogo.png</normaloff>:/images/SpagLogo.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maroon 5</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="icon">
     <iconset resource="qtresource.qrc">
      <normaloff>:/images/Maroon5Logo.png</normaloff>:/images/Maroon5Logo.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>-240</x>
     <y>440</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>392</width>
     <height>236</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmShow">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Show</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_location">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtresource.qrc"/>
 </resources>
 <connections/>
</ui>
